mne/time_frequency/tfr.py

```diff
@@ -266,8 +270,13 @@ def cwt_morlet(X, sfreq, freqs, use_fft=True, n_cycles=7.0, zero_mean=False,
         Number of cycles. Fixed number or one per frequency.
     zero_mean : bool
         Make sure the wavelets are zero mean.
-    decim : int
-        Decimation factor applied after time-frequency decomposition.
+    decim : int | slice
+        To reduce memory usage, slice on the time axis for which the
+        time-frequency decomposition is returned after being calculated
+        for all the time samples.
+        If decim is an integer, it corresponds to the decimation factor
+        similar to slice(None, None, decim).
+        Note that this is brute force decimation, no anti-aliasing is done.
```

Comment:
First describe function when decim is `int`, then when it is `slice`. e.g.

```
Decimation factor after time-frequency decomposition. If `int`,
returns tfr[..., ::decim]. If `slice` returns tfr[..., decim].
```

" Note that this is brute force decimation, no anti-aliasing is done." => Note that decimation may create aliasing artifacts.